### DISCUSSION TITLE
v1.8.1 -Right/Left Split EQ, Audio Signal Visualizations, Volume Mapping

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#888] [General]
* Separate Right/Left channel processing with Expert EQ
* Frequency Volume visualization in Advanced and Expert EQ
* Per App volume visualization in App Mixer
* Volume Remapping - Limit Max Volume or Adjust the Boost amount
* Adjust processing Stop delay
* Improved background CPU usage
* Various Bug Fixes and Improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/'>v1.8.1 -Right/Left Split EQ, Audio Signal Visualizations, Volume Mapping</a>.</em>

**Comment by LeeBinder** [comment]
Hi Roman @nodeful .

Is v1.8.1 still beta = potentially buggy/ not stable, or attributed to the release channel? I'm asking because even though you did not mark it as pre-release here on your GitHun release page as you did with v1.8.0, when I click onto "_Check for Update_" in eqMac v1.7.0, the message is

`You’re up to date! eqMac v1.7.10 is currently the newest version available.`

Only when I check "Beta Program" and click onto "_Check for updates_" again, I get

`A new version of eqMac is available! eqMac v1.8.1 is now available-you have v1.7.10. Would you like to download it now?`

So something doesn't add up here: either v1.8.1 indeed is still beta and you forgot to mark it here @ GitHub, or you forgot to add the v1.8.1 version to the release channel so the eqMac apps can find it as release.

Thanks for eqMac, and for correcting this either way ;)

**Reply by nodeful** [reply]
Hey there :)
You are correct the update feed and GitHub releases are currently not in line and that was done intentionally.
The website serves the latest release on GitHub to new users, so I get more people testing the release before I push it to the wider public.
I would have made v1.8.1 public already but one quite serious bug still persists that I can't replicate at all, so I'm probably going to release v1.8.2 and then push it to everyone.

**Reply by LeeBinder** [reply]
Gotcha, Roman. Makes total sense, I'd do it the same way - good you're handling it the smart way;) .. and good luck leveling out that weird bug 👍
